std/experimental/ndslice/package.d

```diff
@@ -57,6 +57,12 @@ Multidimensional view is presented by $(SUBREF slice, Slice) type.
 ------
 auto matrix = new double[12].sliced(3, 4);
```

Comment:
There is now a fix for this, that will allow (finally!!!) ddoc's unittests to work with module declarations: https://github.com/D-Programming-Language/dmd/pull/5465